From 3114c7f701ee2b4e4e21a599cbf718cea304acbc Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Fri, 21 Dec 2018 20:05:45 -0800
Subject: [PATCH] Remove golang version check

---
 Makefile | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/Makefile b/Makefile
index 32a46b9..8f21bc1 100644
--- a/Makefile
+++ b/Makefile
@@ -19,10 +19,6 @@ else
     SKIP_GO_VERSION_CHECK=
 endif
 
-ifeq ($(SKIP_GO_VERSION_CHECK),)
-    include golang.mk
-endif
-
 ifneq ($(GOPATH),)
     GOARCH=$(shell go env GOARCH)
     ifeq ($(ARCH),)
-- 
2.19.2

